rear_rider_device/test_lidar_proc.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+from pkgutil import get_data
+from sys import stdout
+from ipc.parent_process import ParentProcess
+# import rear_rider_sensors.lidar as lidar
+
+class LidarParentProcess(ParentProcess):
+    def __init__(self):
+        pass
+
+    async def pre_ready(self):
+        pass
+
+    def pre_done(self):
+        pass
+    
+    def no_ack(self):
+        pass
+    
+    def no_on_handler(self):
+        pass
+
+    async def on_get_data(self):
+        data = ("data\n" f"{14} {100}")
+        self.writeline(data)
+
+if __name__ == '__main__':
+    # lidar = lidar.Lidar()
+    lidar_parent_proc = LidarParentProcess()
```

Comment:
Don't miss renaming this!

Reply:
All set [Refactor class and instance names for clarity](https://github.com/Capstone-Projects-2022-Fall/capstone-project-rear-rider/pull/25/commits/602e9524f466add2d387e55dec74470a127395c6)